src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -140,6 +142,13 @@ const NavBar: React.FunctionComponent<PropsType> = ({ redirectionUrl }) => {
   const isBurnTemplate =
     !!venue?.template && IS_BURN && ALL_BURN_TEMPLATES.includes(venue.template);
 
+  const getHeaderLogo = () => {
+    if (venue?.template === VenueTemplate.memrisechats) {
+      return MEMRISE_LOGO_URL;
+    }
+    return isBurnTemplate ? SPARKLEVERSE_LOGO_URL : SPARKLE_LOGO_URL;
```

Comment:
This will likely conflict but should be easy to resolve - `IS_BURN ? SPARKLEVERSE_LOGO_URL : SPARKLE_LOGO_URL`